app/controllers/admin/shelters_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Admin::SheltersController < ApplicationController
+  def index
+    @shelters = Shelter.find_by_sql("SELECT * FROM shelters ORDER BY name DESC;") #order(name: :desc)
```

Comment:
All of the AR and SQL that is taking place here should live at the model level. Then you will just call those methods here in the controller.